doc/classes/LineEdit.xml

```diff
@@ -221,7 +221,7 @@
 		<member name="secret" type="bool" setter="set_secret" getter="is_secret" default="false">
 			If [code]true[/code], every character is replaced with the secret character (see [member secret_character]).
 		</member>
-		<member name="secret_character" type="String" setter="set_secret_character" getter="get_secret_character" default="&quot;*&quot;">
+		<member name="secret_character" type="String" setter="set_secret_character" getter="get_secret_character" default="&quot;•&quot;">
 			The character to use to mask secret input (defaults to "*"). Only a single character can be used as the secret character.
```

Comment:
```suggestion
			The character to use to mask secret input (defaults to "•"). Only a single character can be used as the secret character.
```

Reply:
Should probably remove that bit, it just repeats the generated part that is already visible to the reader. But can be done at some later point, there are probably other cases like that.